### ISSUE TITLE
Using multiple configuration in KUBECONFIG environment breaks eks-node-viewer

### ISSUE THREAD

**Issue by 5nafu** [awslabs/eks-node-viewer#31]
I have multiple configurations in my `KUBECONFIG` environment variable, diveded by a `:` as described in https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#append-home-kube-config-to-your-kubeconfig-environment-variable

Unfortunately, this breaks the eks-node-viewer with the following error:
```
$ eks-node-viewer
2022/12/14 09:08:56 creating client, stat .kubeconfig:/Users/[...]/.kube/config:/Users/[...]/.kube/config.d/01_preferences:/Users/[...]/.kube/config.d/EKS:/Users/[...]/.kube/config.d/local:[...]: no such file or directory
```

**Comment by spmason** [comment]
I noticed this and raised a PR to fix it: https://github.com/awslabs/eks-node-viewer/pull/32

No idea if it'll be accepted, but I'm currently running that build locally and it works for me

**Comment by tzneal** [comment]
Just merged #32, should be fixed now. Thanks @spmason !
